.github/workflows/psalm.yml

```diff
@@ -19,12 +19,31 @@ jobs:
     name: Psalm
     runs-on: Ubuntu-20.04
 
+    env:
+      php-version: '8.1'
+      extensions: json,couchbase,memcached,mongodb,redis,xsl,ldap,dom
+      key: cache-v1 # can be any string, change to clear the extension cache.
```

Comment:
I would inline this in its only usage. It would be more clear than this generic `key` name.